src/classes/window/fullscreen.vala

```diff
@@ -50,13 +50,20 @@ namespace pdfpc.Window {
         protected bool faded_to_black = false;
 
         /**
-         * Fixed layout
+         * Overlay layout. Holds all drawing layers (like the pdf,
+         * the pointer mode etc)
          */
-        protected Gtk.Fixed fixed_layout;
+        protected Gtk.Overlay overlay_layout;
 
-        public void add_to_fixed(Gtk.Widget w, int x, int y) {
-            this.fixed_layout.put(w, x, y);
-        }
+        /**
+         * Drawing area for pointer mode
+         */
+        public Gtk.DrawingArea drawing_surface { get; protected set; }
+
+        /**
+         * Video area for playback
+         */
+        public View.Video video_surface { get; protected set; }
```

Comment:
It is meant that this surface holds all videos (as kind of video layer). How it actually works (with a `Gtk.Fixed`) is an implementation detail :)